dm/dm/master/scheduler/scheduler.go

```diff
@@ -2604,12 +2498,12 @@ func (s *Scheduler) RemoveLoadTask(task string) error {
 // return (worker, source) that is used by transferWorkerAndSource, to try to resolve a paused load task that the source can't be bound to the worker which has its dump files.
 // worker, source	This means a subtask finish load stage, often called by handleLoadTaskDel.
 // worker, ""		This means a free worker online, often called by tryBoundForWorker.
-// "", source		This means a unbounded source online, often called by tryBoundForSource.
+// "", source		This means a source online, often called by tryBoundForSource.
 func (s *Scheduler) getNextLoadTaskTransfer(worker, source string) (string, string) {
```

Comment:
we can also check returned `w.baseInfo.Name` does not equal `worker` because now worker may has multiple bound sources.